### ISSUE TITLE
grafana table JSON view no longer renders json

### ISSUE THREAD

**Issue by vbichov** [grafana/grafana#48843]
**What happened**:
Since grafana 8.4.4 (tested on 8.4.5 and 8.5.2)  table pannel's JSON view Cell display mode no longer renders nice-looking JSON when hovering over the cell

**What you expected to happen**:
JSON view Cell display mode  to show pretty (collapsed) JSON when hovering over the cell

**How to reproduce it (as minimally and precisely as possible)**:
1. create a table pannel 
2. select any datasouce that can return json structures 
3. change `Cell display mode` to `JSON view`

**Anything else we need to know?**:
grafana 8.4.4 working (expected result):
<img width="1772" alt="Screen Shot 2022-05-08 at 11 53 39" src="https://user-images.githubusercontent.com/4352719/167289125-50735bc9-4c57-42e7-b2c3-c970e8be6cd7.png">

grafana 8.5.2 not working (mouse hovering over the cell):
<img width="1780" alt="Screen Shot 2022-05-08 at 11 54 41" src="https://user-images.githubusercontent.com/4352719/167289146-7eaa7bd4-88f2-4f20-9a37-9b4894d3f84e.png">

**Environment**:
- Grafana version: 8.5.2
- Data source type & version: mysql
- OS Grafana is installed on: linux
- User OS & Browser: chrome
- Grafana plugins: not relevant
- Others: for reproduction I used MySQL with the following query:
```
SELECT '{"glossary": {"title": "example glossary","GlossDiv": {"title": "S","GlossList": {"GlossEntry": {"ID": "SGML","SortAs": "SGML","GlossTerm": "Standard Generalized Markup Language","Acronym": "SGML","Abbrev": "ISO 8879:1986","GlossDef": {"para": "A meta-markup language, used to create markup languages such as DocBook.","GlossSeeAlso": ["GML", "XML"]},"GlossSee": "markup"}}}}}' AS data
UNION SELECT '{"some": "data"}' 
```

**Comment by zuchka** [comment]
perhaps this PR is relevant? the timing is right

[#46020](https://github.com/grafana/grafana/pull/46020)

**Comment by dprokop** [comment]
@vbichov the tooltip preview is now gone in favor of cell inspect that's documented here: https://grafana.com/docs/grafana/latest/visualizations/table/#cell-value-inspect

**Comment by pprkut** [comment]
Which version did the fix go into?

**Comment by mauriciotrooh** [comment]
@pprkut do you know which version the fix went into? have you tested if the #46020 fixes the issue? Thanks!

**Comment by pprkut** [comment]
@mauriciotrooh The "fix" that @dprokop mentioned went into 9.0.0, I just misread it initially. I was looking for something that restored the old behavior. `cell value inspect` works very different from what was there before, and while it is better in some situations, it's worse in others. That's why I opened a discussion about it in #52540.
